travis.yml.template

```diff
@@ -93,11 +93,13 @@ jobs:
         - git diff --stat --exit-code .travis.yml
 
     - stage: Deploy
-      script:
+      before_script:
         - |
-          if [ "false" != "$TRAVIS_PULL_REQUEST" ]; then
```

Comment:
This can also be done through https://docs.travis-ci.com/user/conditional-builds-stages-jobs/#conditional-jobs

Reply:
https://github.com/nodejs/docker-node/blob/master/.travis.yml#L39

Reply:
This was deleted ... I didn't see the point here.